services/checks/checks.go

```diff
@@ -96,6 +102,10 @@ type Service struct {
 // New returns Service with given PMM version.
 func New(agentsRegistry agentsRegistry, alertsRegistry alertRegistry, db *reform.DB, pmmVersion string) *Service {
 	l := logrus.WithField("component", "checks")
+
+	pmmAgentVersionTwoDotSix = mustParseVersion("2.6.0")
+	pmmAgentVersionTwoDotSeven = mustParseVersion("2.7.0")
```

Comment:
We shouldn't modify a global state in a function. Either use a Service field, or `init()`.

Reply:
`init` then. To be honest, it's exactly how I want to initialize those variables at first, but we have strange linter that asks to not use `init` functions.